evaluation/ragas_eval.py

```
@@ -0,0 +1,228 @@
+            result = ragas_evaluate(dataset, metrics=metrics)
+            result_df = result.to_pandas()
+            for df_idx, (_, row) in enumerate(valid):
+                if df_idx < len(result_df):
+                    for m in _RAGAS_METRICS:
+                        val = result_df.iloc[df_idx].get(m)
+                        row[f"{prefix}_ragas_{m}"] = (
+                            round(float(val), 4)
+                            if val is not None and val == val else None
+                        )
+            print(f"[RAGAS] {prefix}: 완료")
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)